public/index.html

```diff
@@ -29,11 +35,28 @@
     ></script>
 
     <link href="css/home.css" rel="stylesheet" />
-    <script defer src="/js/common.js"></script>
-    <script defer src="js/home.js"></script>
   </head>
   <body>
-    <header></header>
+    <header class="sticky-top nav-bar"></header>
+    <div class="container-fluid cover min-vh-100 p-0">
+      <div class="row justify-content-center m-0">
+        <div
+          class="col-12 col-md-6 text-light d-flex justify-content-end align-items-center py-5 fs-2"
+        >
+          <p>
+            <span>Ready to level up your device?</span>
+            <br />
+            <span class="fw-bold">
+              LETS BUILD YOUR DREAM PC <br />
+              TOGETHER!
+            </span>
+          </p>
+        </div>
+        <div class="col-12 col-md-6 d-flex">
+          <img src="./img/home/custom_pc-s.png" class="m-auto w-75" alt="" />
```

Comment:
absolute path would be nice
```suggestion
          <img src="/img/home/custom_pc-s.png" class="m-auto w-75" alt="" />
```